spec/GraphQLOverHTTP.md

```diff
@@ -201,8 +201,38 @@ may not be the GraphQL server but instead some intermediary such as API gateways
 
 ## Status Codes
 
-If the `data` entry in the response has any value other than `null` (when the operation has successfully executed
-without error) then the response should use the 200 (OK) status code.
-If the operation failed before or during execution, due to a syntax error, missing information, validation error
-or any other reason, then response should use the 4XX or 5XX status codes.
-It is recommended to use the same error codes as the [reference implementation](https://github.com/graphql/express-graphql).
+If the response has Content-Type GraphQL and contains a non-null `data` entry,
+then it MUST have status code `2xx`, and it SHOULD have status code `200`.
+
+If the response has Content-Type GraphQL and has status code `2xx`,
+the entry `data` entry must be either:
+- equal to `null` (in case of an execution error)
+- not present (in case of a validation error)
```

Comment:
This is wrong, these rules apply to non-`2xx` status codes. Suggest rewording to:

```suggestion
If the response has Content-Type GraphQL and has a non-success
status code (i.e. `4xx` or `5xx`), the entry `data` entry must be
either:
- equal to `null` (in case of an execution error)
- not present (in case of a validation error)
```

Reply:
This section is also in https://github.com/graphql/graphql-over-http/pull/79, which this PR depends upon.

Reply:
[`9e75abc` (#79)](https://github.com/graphql/graphql-over-http/pull/79/commits/9e75abccfb0030dbbddf324b5fa621bf37b8e919)